routes/index.js

```diff
@@ -415,6 +415,55 @@ var routes = {
         reply('Dispute closed');
       }
     );
+  },
+  'stripe-charge-succeeded': function(request, reply) {
+    var event = request.payload;
+    var charge = event.data.object;
+
+    if (event.type !== 'charge.succeeded') {
+      return reply('This hook only processes charge succeeded events');
+    }
+
+    stripe.retrieveCharge(
```

Comment:
I make this call to expand the invoice object, which then gives me the subscription id, which I can request.